2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,2676 @@
+          description: Timestamp (in ISO 8601 format) when the entity was created.
+          title: Creation Date
+          type: string
+        description:
+          description: Description of an entity.
+          title: Description
+          type: string
+        has_anatomical_entity:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/AnatomicalEntity'
+          description: The Anatomical entity, that represents the site, from which
+            the             Biospecimen was retrieved. Typically, a concept from             Uber-anatomy
+            Ontology (UBERON). For example, 'UBERON:0008307'             indicates
+            that the Biospecimen was extracted from the 'Heart Endothelium'             of
+            an Individual.
+          title: Has Anatomical Entity
+        has_disease:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/Disease'
+            type: array
+          description: The Disease entity that is associated with the Individual.             Typically,
+            a concept from Mondo Disease Ontology. For example,             'MONDO:0005267'
+            indicates that the Individual suffers from             'Heart Disease'.
+          title: Has Disease
+        has_individual:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/Individual'
+          description: The Individual entity from which this Biospecimen was derived.
+          title: Has Individual
+        has_phenotypic_feature:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/PhenotypicFeature'
+            type: array
+          description: The Phenotypic Feature entity that is associated with the             Individual.
+            Typically, a concept from Human Phenotype Ontology.             For example,
+            'HP:0100244' indicates that the Individual exhibits             'Fibrosarcoma'
+            as one of its phenotype.
+          title: Has Phenotypic Feature
+        id:
+          description: An identifier that uniquely represents an entity.
+          title: Id
+          type: string
+        name:
+          description: The name for an entity.
+          title: Name
+          type: string
+        update_date:
+          description: Timestamp (in ISO 8601 format) when the entity was updated.
+          title: Update Date
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: Biospecimen
+      type: object
+    CreateDataset:
+      properties:
+        has_data_access_policy:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/DataAccessPolicy'
```

Comment:
Specifying a string here cannot work currently because we don't have a `POST /data_access_policy` endpoint, yet. I would suggest to remove the ambivalence here and for now and only go with the second option. If we later decide we need the additional endpoint `POST /data_access_policy`, we can still change it. (Introduce complexity when it's needed)

Reply:
Thanks for pointing that out. I think it would be much straight forward to have a `POST /data_access_policy` endpoint to support creation of a DAP. This would also over the user journey outlined in the specification. Similarly for DAC.